libethereum/TransactionQueue.cpp

```diff
@@ -348,6 +348,7 @@ void TransactionQueue::clear()
 	WriteGuard l(m_lock);
 	m_known.clear();
 	m_current.clear();
+	m_dropped.clear();
```

Comment:
This looks good.

Reply:
This is fine, as this method is used only by `ClientTest` I think

Reply:
No. this fixes a bug. if transaction is considered invalid but then after rewind it is being imported again it might become valid.

Reply:
Right, and rewind is used only in test mode